openfl/interface/interactive_api/experiment.py

```diff
@@ -71,17 +75,24 @@ def get_last_model(self):
 
         return self._rebuild_model(tensor_dict)
 
-    def _rebuild_model(self, tensor_dict):
+    def _rebuild_model(self, tensor_dict, called_from_get_best_model=False):
```

Comment:
The new argument name assumes that this method that the internal logic of the method depends on external method names. Better to rename to something that actually means a parameter to _rebuild_model method.